grapher/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -79,28 +81,38 @@ export class ScatterPlotChart
     @observable private hoverColor?: Color
 
     transformTable(table: OwidTable) {
+        if (this.manager.excludedEntities) {
+            const excludedEntityIdsSet = new Set(this.manager.excludedEntities)
```

Comment:
sidenote: post merge as part of a grapher config cleanup we should probably change `excludedEntities` from Ids to entityNames.